packages/xo-server/src/xapi/index.js

```diff
@@ -405,7 +404,7 @@ export default class Xapi extends XapiBase {
       return await this.call('VM.copy', snapshot ? snapshot.$ref : vm.$ref, nameLabel, sr ? sr.$ref : '')
     } finally {
       if (snapshot) {
-        await this._deleteVm(snapshot)
+        await this.deleteVm(snapshot)
```

Comment:
Use `VM_destroy` everywhere it's easy to do so.

And remove `deleteVm` if no one is using it anymore.